### PR TITLE
Fix build error for the rstudio image

### PR DESCRIPTION
**Pull request by Jswig**
The RStudio image does not have access to the `devtools` package to install `containerit` from Github. Fixed this to install and use `remotes` instead

### CODE REVIEW

**Comment by ivotron** [comment]
can you please rebase master?

**Comment by ivotron** [comment]
also, you can avoid having to reconcile changes on the upstream's master branch with your fork's branch (which usually differ significantly due to squashed commits) by working on feature branches and open PRs from there. So you can first pull upstream changes to your local repo, then create a branch locally, and then push that branch to github and open a PR from there.

**Comment by Jswig** [comment]
> 
> 
> can you please rebase master?
This did not have any effect on my side. Am I missing something?

File: r/rstudio/Dockerfile [hunk 1]
```diff
@@ -5,12 +5,12 @@ LABEL description="Used as a base for Popper workflows using R, rstudio and the
 # labels match the corresponding Rocker image
 LABEL version="4.0.0" 
 
-
 # move configuration file where rstudio server will find it
 COPY rsession.conf /etc/rstudio 
 
 # install containerit
-RUN R -e "devtools::install_github(\"o2r-project/containerit\")"
+RUN R -e "install.packages(\"remotes\")" \
```
Comment (ivotron) [inline r/rstudio/Dockerfile:12]: this is missing an `&&` at the end

**Comment by ivotron** [comment]
> This did not have any effect on my side. Am I missing something?

sorry, I think I got confused. I thought there was a merge conflict with master but there isn't one

File: r/rstudio/Dockerfile (hunk 1 above)
Comment (Jswig) [reply r/rstudio/Dockerfile]: Fixed. This should be ready to merge

**Pull request merged**